src/viking.js

```diff
@@ -1,11 +1,56 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health = health;
+        this.strength = strength;
+    }
+    attack() {
+        return this.strength;
+    }
+    receiveDamage(damage) {
+        this.health -= damage;
+    }
+}
+
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name, health, strength) {
+        super(health, strength);
+        this.name = name;
+    }
+    receiveDamage(damage) {
+        this.health -= damage;
+        if (this.health > 0) {
+            return (`${this.name} has received ${this.damage} points of damage`)
+        } else {
+            return (`${this.name} has died in act of combat`)
+        }
+    }
```

Comment:
Good work, I just wanted to point out that because you have an `if(){...} else{...}`, you can use a ternary operator to make your code shorter
```
return this.health > 0 ? `${this.name} has received ${this.damage} points of damage`: `${this.name} has died in act of combat`;